src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void addTaskTest() {
```

Comment:
Would be nice if logical units of code were separated in this method, just like in ParserTest.java